tests/array/tarray.nim

```diff
@@ -587,3 +587,10 @@ block t12466:
     a[0'u16 + i] = i
   for i in 0'u16 ..< 8'u16:
     a[0'u16 + i] = i
+
+block t17705:
+  # https://github.com/nim-lang/Nim/pull/17705
+  var a = array[0, int].low
+  a = int(a)
+  var b = array[0, int].high
```

Comment:
Please use `doAssert`

```nim
block t17705:
  # https://github.com/nim-lang/Nim/pull/17705
  var a = array[0, int].low
  a = int(a)
  doAssert a == 1
  var b = array[0, int].high
  b = int(b)
  doAssert b == -1
```

Reply:
I'm not sure that's very relevant, since the goal of this test is to see if the code compiles, and the value of a & b doesn't change in this code
But I could do it, sure. a would be 0 though not 1